docs/pages/versions/unversioned/guides/push-notifications.md

```diff
@@ -15,7 +15,7 @@ In order to send a push notification to somebody, we need to know about their de
 ![Diagram explaining saving tokens](/static/images/saving-token.png)
 
 ```javascript
-import { Permissions } from 'expo';
+import { Permissions,Notifications } from 'expo';
```

Comment:
Thanks for fixing it! 😄

I think since we already `import * as Permissions from 'expo-permissions';` below, we only need to ` import { Notifications } from 'expo';` here.